sdk/Lusid.Sdk.Tests/tutorials/Instruments/CreditDefaultSwapExamples.cs

```diff
@@ -175,5 +176,90 @@ public void CreditDefaultSwapGetPortfolioCashFlowsExample(ModelSelection.ModelEn
             var cds = InstrumentExamples.CreateExampleCreditDefaultSwap();
             CallLusidGetPortfolioCashFlowsEndpoint(cds, model);
         }
+
+        [TestCase("2020-06-05T00:00:00.0000000+00:00")] // calculate upfront charge for cds contract
+        public void CreditDefaultSwapIsdaCdsValuationExample(string sTestNow)
+        {
+            // Purpose: Demo the valuation of a credit default swap using the IsdaCds vendor library via Lusid
+            var testScope = Guid.NewGuid().ToString();
+            var testNow = DateTimeOffset.Parse(sTestNow);
+            var cds = InstrumentExamples.CreateExampleCreditDefaultSwap() as CreditDefaultSwap;
+
+            // CREATE the configuration Recipe
+            var pricingOptions = new PricingOptions();
+            // tell Lusid that CDS instruments should be valued with the default model in IsdaCds
+            var modelRules = new List<VendorModelRule>
+                {new VendorModelRule(VendorModelRule.SupplierEnum.IsdaCds, "VendorDefault", "CreditDefaultSwap")};
+            var pricingContext = new PricingContext(options: pricingOptions, modelRules: modelRules);
+            // tell Lusid where to find data for IsdaCreditCurves; the IsdaYieldCurve is found according to a default rule
```

Comment:
can we make the rules explicit. I'd argue for adding a couple of lines explaining what they do in this context. That can help this example be more useful for our sales engineers and clients.

Reply:
done